### PR TITLE
Source Google Ads: Add `audience` and `user_interest` streams

### LINKED ISSUE #24750: Source Google Ads: Include audience and user_interest to full-refresh ingestion
## Tell **us about the problem you're trying to solve**
I would like to have the access of the audience and user_interest of Google Ads. Basically, I need to extract information from Google Ads to do a analysis. The audience table present the user_interest ids, but the name is not available there but it is available on user_interests. Next I present the google documentation: 

* Audience: https://developers.google.com/google-ads/api/fields/v13/audience
* User_Interest: https://developers.google.com/google-ads/api/fields/v13/user_interest

## Describe the solution you’d like
I need extract information from audience and its users_intererts name, both on full-refresh mode.

## Are you willing to submit a PR?
Yes.

### PR DESCRIPTION
**Pull request by lazebnyi**
## What
Fixed: https://github.com/airbytehq/airbyte/issues/24750

Need implement: 

- Audience: https://developers.google.com/google-ads/api/fields/v13/audience

- User_Interest: https://developers.google.com/google-ads/api/fields/v13/user_interest

## How
Implemented audience and users_intererts streams with full refresh. 

## Pre-merge Actions

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-google-ads test report (commit [`3ab83e2b9c`](https://github.com/airbytehq/airbyte/pull/27959/commits/3ab83e2b9ce2dabb4d1e30566b12a992f8017951)) - ❌

⏲️  Total pipeline duration: 01mn54s 

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-ads/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| Connector version increment check                                        | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-google-ads docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/lazebnyi_24750-add-audience-and-user-interest-streams/1688515369/3ab83e2b9ce2dabb4d1e30566b12a992f8017951/source-google-ads/0.4.3/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-ads test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-ads test report (commit [`481615974a`](https://github.com/airbytehq/airbyte/pull/27959/commits/481615974a64349cd3ab526179dfd40982f8dce4)) - ✅

⏲️  Total pipeline duration: 16mn09s 

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-ads/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-google-ads docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Acceptance tests                                                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/lazebnyi_24750-add-audience-and-user-interest-streams/1688555094/481615974a64349cd3ab526179dfd40982f8dce4/source-google-ads/0.4.3/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-ads test
```

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/audience.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      ]
+    },
+    "audience.status" : {
+      "type" : [
+        "null",
+        "string"
+      ]
+    }
+  }
+}
```
Comment (bazarnov) [inline airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/audience.json:48]: new line is missed

**Review by bazarnov — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-google-ads test report (commit [`1fb9bd6548`](https://github.com/airbytehq/airbyte/pull/27959/commits/1fb9bd6548db3c37358627c856c70fbced5e1943)) - ❌

⏲️  Total pipeline duration: 32mn49s 

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-ads/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-google-ads docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Acceptance tests                                                         | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/lazebnyi_24750-add-audience-and-user-interest-streams/1688657028/1fb9bd6548db3c37358627c856c70fbced5e1943/source-google-ads/0.4.3/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-ads test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-ads test report (commit [`733a1b19d9`](https://github.com/airbytehq/airbyte/pull/27959/commits/733a1b19d9886c010b31603a8d32707874071f28)) - ❌

⏲️  Total pipeline duration: 4.76s 

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-ads/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-google-ads docker image for platform linux/x86_64           | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/lazebnyi_24750-add-audience-and-user-interest-streams/1688731415/733a1b19d9886c010b31603a8d32707874071f28/source-google-ads/0.4.3/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-ads test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-ads test report (commit [`733a1b19d9`](https://github.com/airbytehq/airbyte/pull/27959/commits/733a1b19d9886c010b31603a8d32707874071f28)) - ❌

⏲️  Total pipeline duration: 4.50s 

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-ads/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-google-ads docker image for platform linux/x86_64           | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/lazebnyi_24750-add-audience-and-user-interest-streams/1688731648/733a1b19d9886c010b31603a8d32707874071f28/source-google-ads/0.4.3/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-ads test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-ads test report (commit [`733a1b19d9`](https://github.com/airbytehq/airbyte/pull/27959/commits/733a1b19d9886c010b31603a8d32707874071f28)) - ❌

⏲️  Total pipeline duration: 4.55s 

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-ads/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-google-ads docker image for platform linux/x86_64           | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/lazebnyi_24750-add-audience-and-user-interest-streams/1688731924/733a1b19d9886c010b31603a8d32707874071f28/source-google-ads/0.4.3/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-ads test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-ads test report (commit [`733a1b19d9`](https://github.com/airbytehq/airbyte/pull/27959/commits/733a1b19d9886c010b31603a8d32707874071f28)) - ❌

⏲️  Total pipeline duration: 4.42s 

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-ads/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-google-ads docker image for platform linux/x86_64           | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/lazebnyi_24750-add-audience-and-user-interest-streams/1688732122/733a1b19d9886c010b31603a8d32707874071f28/source-google-ads/0.4.3/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-ads test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-ads test report (commit [`f285534bc5`](https://github.com/airbytehq/airbyte/pull/27959/commits/f285534bc5b3aba9f8751289f246acc48f580b16)) - ✅

⏲️  Total pipeline duration: 14mn04s 

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-ads/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-google-ads docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Acceptance tests                                                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/lazebnyi_24750-add-audience-and-user-interest-streams/1688732388/f285534bc5b3aba9f8751289f246acc48f580b16/source-google-ads/0.4.3/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-ads test
```

**Pull request merged**